master/internal/api_experiment.go

```diff
@@ -1132,7 +1133,15 @@ func (a *apiServer) MetricNames(req *apiv1.MetricNamesRequest,
 	var timeSinceLastAuth time.Time
 	var config expconf.ExperimentConfig
 	var searcherMetric string
+	// r := rand.New()
 	for {
+
+		n := rand.Float32()
+		fmt.Println(n)
+		if n > 0.5 {
+
+			return errors.New("error looking up experiment state")
+		}
```

Comment:
this is for debugging purposes, to test what happens if an error is returned mid-stream